src/net.h

```diff
@@ -586,6 +587,9 @@ class CConnman
 
     std::atomic<int64_t> m_next_send_inv_to_incoming{0};
 
+    //! -bind=<address>:<port>=onion arguments.
```

Comment:
- this documentation is good to provide but a bit succinct; maybe also describe what `m_onion_binds` is
- could use the `/**` doxygen formatting like `m_try_another_outbound_peer` just above

Reply:
[Developer Notes](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-doxygen-compatible-comments) suggest `//!` style to describe a member.

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/19998#issuecomment-703090033).